scripts/NFTVesting/deploy.js

```diff
@@ -0,0 +1,71 @@
+const { ethers } = require("hardhat");
+
+// Test script for deploying the contract
+
+async function main() {
+	// Local Blockchain Deployment
+	const [owner, addr1] = await ethers.getSigners();
+
+	// Deploying Basic NFT Contract
+	const basicNFTContract = await ethers.getContractFactory("BasicNft");
+	const basicNFTInstance = await basicNFTContract.deploy("TestNft", "TFT");
+
+	// Minting NFT
+	const basicMintTx = await basicNFTInstance.mintNft();
+	await basicMintTx.wait(1);
+	console.log(
+		`Basic NFT index 0 tokenURI: ${await basicNFTInstance.tokenURI(0)}`
+	);
+
+	const nftOwner = await basicNFTInstance.ownerOf(0);
+	console.log(`nftOwner: ${nftOwner.address}`);
+
+	// getting timestamp
+	const blockNumBefore = await ethers.provider.getBlockNumber();
+	const blockBefore = await ethers.provider.getBlock(blockNumBefore);
+	const timestampBefore = blockBefore.timestamp;
+	console.log("timestampBefore: ", timestampBefore);
+
+	// Deploying Timelock
+	const timeLock = await ethers.getContractFactory("BasicNFTTimelock");
+	const timeLockInstance = await timeLock.deploy(
+		basicNFTInstance.address,
```

Comment:
Not very sure what the input is for 
```
    constructor(
        IERC721 nft_,
        uint256 tokenId_,
        address beneficiary_,
        uint256 releaseTime_
        ```
        
        Might need some help @junhaoyap

Reply:
Pretty sure it should be the basicNFTInstance itself - did that not work?